[RHACS] ROX-21146: Adding the 4.4 Release notes

agantony opened the pull request:
<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS#<jira-issue-id>]: <short-description-of-the-pr> --->

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):
4.4
<!--- Specify the version or versions of OpenShift your PR applies to. -->

Issue:
https://issues.redhat.com/browse/ROX-21146
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

Link to docs preview:
https://72699--docspreview.netlify.app/openshift-acs/latest/release_notes/44-release-notes
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

SME review:
- [x] SME has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
- Merge into `rhacs-docs` and cherry-pick to:
    - `rhacs-docs-4.4`
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

agantony commented:
/label RHACS

fredrb commented:
@agantony I've added the suggestion as a comment yesterday. Could you please check and let me know if it's sufficient?

agantony reviewed (commented):
Left a few comments.

agantony commented:
> @agantony I've added the suggestion as a comment yesterday. Could you please check and let me know if it's sufficient?

@fredrb Thanks, I've made a few adjustments and added a new entry under "Bug fixes".
PS: https://github.com/openshift/openshift-docs/pull/72699/files#:~:text=*%20Fixed%20an,24%20MB.

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+
+* By requesting the `central-monitoring-tls` \ `sensor-monitoring-tls` secrets at startup, you can ensure the presence of the {osp} monitoring `/metrics` server certificate. This requirement only applies if {osp} Monitoring is enabled.
+
+* Simplify your configuration by using `ROX_MEMLIMIT` instead of `GOMEMLIMIT` in configuration files. You can now capture your deployment memory limits and easily adjust `GOMEMLIMIT` by using `ROX_MEMLIMIT`. Define `ROX_MEMLIMIT` as an integer representing the number of bytes and simplify your configuration for more efficient deployment.
```
I don't think this quite captures the fact that the `GOMEMLIMIT` environment variable is going away and being replaced by `ROX_MEMLIMIT`.  Also I would like to add a sentence about manual roxctl upgrades here. Here are my edits - maybe @RTann can review?

```suggestion
* The `ROX_MEMLIMIT` environment variable replaces the `GOMEMLIMIT` variable in configuration files. You can now capture your deployment memory limits by using the `ROX_MEMLIMIT` variable. Define `ROX_MEMLIMIT` as an integer representing the number of bytes and simplify your configuration for more efficient deployment. If you are manually upgrading {product-title-short} by using the `roxctl` CLI, you must manually edit the deployments to use the new variable. See xref:../upgrading/upgrade-roxctl.adoc#upgrade-roxctl[upgrading/upgrade-roxctl] for more information.
```

RTann replied:
`GOMEMLIMIT` is a standard Go environment variable, which is still acceptable to use, but we replaced all instances of it in our configuration files with `ROX_MEMLIMIT` so we can do some extra stuff. Maybe we can make it clear that `GOMEMLIMIT` is still ok, but `ROX_MEMLIMIT` is preferred

agantony replied:
s/ Use the `ROX_MEMLIMIT`environment variable, which replaces the `GOMEMLIMIT` variable in configuration files. Although you can still use `GOMEMLIMIT` as a standard Go environment variable, use `ROX_MEMLIMIT` instead to more efficiently capture the memory limits for your deployment. Define `ROX_MEMLIMIT` as an integer representing the number of bytes to simplify your configuration. If you are manually upgrading {product-title-short} by using the `roxctl` CLI, ensure to edit the deployments manually to use the new variable. 

For more information, see xref:../upgrading/upgrade-roxctl.adoc#upgrade-roxctl[Manually upgrading using the roxctl CLI].


WDYT? @RTann @kcarmichael08

RTann replied:
> Although you can still use `GOMEMLIMIT` as a standard Go environment variable, use `ROX_MEMLIMIT` instead to more efficiently capture the memory limits for your deployment. Define `ROX_MEMLIMIT` as an integer representing the number of bytes to simplify your configuration.

Although you may still use the standard Go environment variable `GOMEMLIMIT`, use `ROX_MEMLIMIT` instead to more effectively capture the memory limits for your deployment. `ROX_MEMLIMIT` will set the "soft" memory limit of the Go process to 95% of the configured amount. Define `ROX_MEMLIMIT` as an integer without any units representing the number of bytes.

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+{product-title-short} 4.4 includes the following new features, improvements, and updates:
+
+Compliance::	
+* xref:../release_notes/44-release-notes.adoc#rhacs-new-compliance-capabilities_release-notes-44[{product-title-short} new Compliance capabilities (Technology Preview)]
```
```suggestion
* xref:../release_notes/44-release-notes.adoc#rhacs-new-compliance-capabilities_release-notes-44[New Compliance capabilities (Technology Preview)]
```

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+====
+* In {product-title-short} 4.4, both the StackRox Scanner and the new Vulnerability Scanner V4 are available for scanning workloads, and the existing StackRox Scanner reports node and platform vulnerabilities.
+* {product-title-short} upgrades and new installations use the StackRox Scanner by default.
+* You now have the option to choose the new Vulnerability Scanner V4 instead of default StackRox Scanner, which offers additional benefits and an extended scope.
```
This is not true - please confirm @RTann 

```suggestion
* You now have the option to enable the new Vulnerability Scanner V4 in addition to the default StackRox Scanner, which offers additional benefits and an extended scope.
```

RTann replied:
The phrasing you're suggesting sounds right. Maybe we should make it clear Scanner V4 is only for image scanning at this time while StackRox Scanner is still needed for node and platform scanning

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+* xref:../release_notes/44-release-notes.adoc#authentication-of-aws-and-gcp-integrations-by-using-short-lived-tokens_release-notes-44[Authentication of AWS and GCP integrations by using short-lived tokens (Technology Preview)]
+
+Vulnerability Management::
+* xref:../release_notes/44-release-notes.adoc#vulnerability-scanner-v4-that-utilizes-upstream-claircore_release-notes-44[Vulnerability Scanner V4 that utilizes upstream ClairCore]
```
just Scanner V4

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+This release adds improvements related to the following components and concepts:
+
+[id="vulnerability-scanner-v4-that-utilizes-upstream-claircore_{context}"]
+=== Vulnerability Scanner V4 that utilizes upstream ClairCore 
```
just Scanner V4

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+
+* *Consistent and accurate scanning*: Reliable vulnerability scan results across the entire Red Hat product ecosystem, Red Hat {product-title-short}, and Red Hat Quay.
+* *Expanded language and operating system support*: Expanded support for Golang in language vulnerability scanning and inclusion of Oracle Linux, SUSE Linux Enterprise, and Photon OS in operating system scanning.
+* *Comprehensive vulnerability database source*: Adoption of `OSV.dev` as the vulnerability database source for all supported programming language packages.
```
maybe link it: https://osv.dev/

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+* In {product-title-short} 4.4, both the StackRox Scanner and the new Vulnerability Scanner V4 are available for scanning workloads, and the existing StackRox Scanner reports node and platform vulnerabilities.
+* {product-title-short} upgrades and new installations use the StackRox Scanner by default.
+* You now have the option to choose the new Vulnerability Scanner V4 instead of default StackRox Scanner, which offers additional benefits and an extended scope.
+* It is anticipated that the new Vulnerability Scanner V4 will be the default Scanner in a future release.
```
just Scanner V4

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,560 @@
+* "StackRox Scanner settings" in xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#scanner-settings_install-central-config-options-ocp[Installing {product-title-short} on {osp}]
+* "Scanner V4" in xref:../installing/installing_other/install-central-other.adoc#central-services-public-configuration-file-scannerv4_install-central-other[Installing {product-title-short} on other platforms]
+
+Scanner V4 has additional memory and storage requirements in addition to the memory and storage needed to run the default StackRox scanner. 
```
StackRox Scanner (last word in the sentence)

agantony commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,558 @@
+{product-title-short} 4.4 offers the following new Compliance capabilities:
+
+* A seamless experience that combines Compliance Operator and {product-title-short}.
+* You can configure, schedule and run infrastructure scans and review the results directly from {product-title-short}.
```
Need info pending.

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+* If you are offline, you no longer need to update the Collector support packages.
+By avoiding issues related to eBPF probe retrieval, such as the risk of losing the network connection or if the probe is not present, you can update a cluster smoothly.
+
+[id="bring-your-own-database-for-central-database-is-now-generally-available_{context}"]
+=== Bring your own database for Central database is now generally available
+
+With {product-title-short} 4.4, you can now use your own PostgreSQL-compatible database for the {product-title-short} Central database, giving you the choice to deploy it either within or outside the cluster. Whether on bare metal, virtual machines or as a cloud-hosted service, you can tailor the deployment to your specific requirements. To ensure optimal performance, you must run the database in ...
+
+For more information about the supported platforms, see the Red Hat Knowledgebase solution link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix].
```
```suggestion
[id="bring-your-own-database-for-central-is-now-generally-available_{context}"]
=== Bring your own database for RHACS Central is now generally available

With {product-title-short} 4.4, you can now use your own PostgreSQL-compatible database with {product-title-short} Central, allowing you to deploy it within or outside the cluster. Whether on bare metal, virtual machines, or as a cloud-hosted service, you can tailor the deployment to your specific requirements. To ensure optimal performance, you must run the database close to the {product-title-short} Central services.

For more information, see xref:../../installing/installing_ocp/install-central-ocp.adoc#install-central-operator-external-db_install-central-ocp[Installing Central with an external database using the Operator method].
```

**nitpick:** I've updated the language to match waht we used in previous releases and added a link to the docs.

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations*, and then select the integration type, for example, *Amazon ECR*. Select an existing integration. Alternatively, click *New integration* to create a new integration.
+
+In the integration form, select *Use container IAM role* to enable STS role assume that applies to Amazon EKS and {osp} clusters. 
```
**issue:** This should be`AssumeRole`
```suggestion
In the integration form, select *Use container IAM role* to enable STS AssumeRole that applies to Amazon EKS and {osp} clusters. 
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+For Google Artifact Registry, Google Container Registry, Google Security Command Center and Google Cloud Storage integrations, support for authentication through workload identity federation has been added. 
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations*, and then select the integration type, for example, *Google Artifact Registry*. Select an existing integration. Alternatively, click *New integration* to create a new integration.
```
```suggestion
In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations* and select the integration type, for example, *Google Artifact Registry*. Select an existing integration or click *New integration* to create a new one.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations*, and then select the integration type, for example, *Google Artifact Registry*. Select an existing integration. Alternatively, click *New integration* to create a new integration.
+
+In the integration form, select *Use workload identity* to enable token federation that applies to Google GKE and {osp} clusters. 
```
```suggestion
In the integration form, select *Use workload identity* to enable token federation for Google GKE and {osp} clusters. 
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations*, and then select the integration type, for example, *Google Artifact Registry*. Select an existing integration. Alternatively, click *New integration* to create a new integration.
+
+In the integration form, select *Use workload identity* to enable token federation that applies to Google GKE and {osp} clusters. 
+
```
**suggestion:** I've added docs for this one.
```suggestion

For more information, see xref:../../integration/integrate-using-short-lived-tokens.adoc#integrate-using-short-lived-tokens[Integrating RHACS using short-lived tokens].

```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+[id="cluster-discovery-by-using-cloud-source-integrations_{context}"]
+=== Cluster discovery by using cloud source integrations
+
+{product-title-short} 4.4 introduces a new cloud source integration type, which includes integration types for Paladin Cloud and {cloud-redhat-com}. Integrated cloud sources enable {product-title-short} to discover cluster assets from the connected accounts. The discovered clusters are matched against clusters already secured by Central. 
```
```suggestion
{product-title-short} 4.4 introduces a new cloud source integration type, which includes integration types for Paladin Cloud and {cloud-redhat-com}. Integrated cloud sources enable {product-title-short} to discover cluster assets from the connected accounts. {product-title-short} matches the discovered clusters against clusters already secured by Central.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+{product-title-short} 4.4 introduces a new cloud source integration type, which includes integration types for Paladin Cloud and {cloud-redhat-com}. Integrated cloud sources enable {product-title-short} to discover cluster assets from the connected accounts. The discovered clusters are matched against clusters already secured by Central. 
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters -> Discovered clusters* to view the current status of the discovered clusters.
```
```suggestion
In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters -> Discovered clusters* to view the current status of the discovered clusters.

For more information, see xref:../../integration/integrate-with-cloud-management-platforms.adoc#integrate-with-cloud-management-platforms[Integrating with cloud management platforms]

```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+{product-title-short} 4.4 introduces the option of issuing short-lived API tokens for interaction with the Central API.
+
+You can exchange the short-lived API tokens for an OpenID Connect (OIDC) identity token. This is useful for the authentication and authorization of machines in, for example, CI environments.
```
**nitpick:** 
```suggestion
You can exchange the short-lived API tokens for an OpenID Connect (OIDC) identity token. This is useful for authenticating and authorizing machines in CI environments, for example.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+[id="migration-to-stock-red-hat-openshift-sccs-during-manual-upgrade-by-using-roxctl-cli_{context}"]
+=== Migration to stock {osp} SCCs during manual upgrade by using roxctl CLI
+
+With {product-title-short} 4.4, the platform moves away from using custom security context constraints (SCCs). Instead, {product-title-short} services use {osp} SCCs, which ensures future proof operation and a consistent security posture for {product-title-short}.
```
```suggestion
With {product-title-short} 4.4, the platform moves away from using custom security context constraints (SCCs). Instead, {product-title-short} services use {osp} SCCs, which ensures future-proof operation and a consistent security posture for {product-title-short}.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* The default memory requirement for `scanner-db` has been increased from 200MiB to 512MiB to prevent out of memory (OOMs) errors during database initialization when the memory pressure reaches the node.
```
```suggestion
* The default memory requirement for `scanner-db` has been increased from 200MiB to 512MiB to prevent out-of-memory (OOM) errors during database initialization when the memory pressure reaches the node.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+* By requesting the `central-monitoring-tls` \ `sensor-monitoring-tls` secrets at startup, you can ensure the presence of the {osp} monitoring `/metrics` server certificate. This requirement only applies if {osp} Monitoring is enabled.
+
+* Use the `ROX_MEMLIMIT` environment variable, which replaces the `GOMEMLIMIT` variable in configuration files. Although you can still use the standard Go environment variable `GOMEMLIMIT`, you should use `ROX_MEMLIMIT` instead to more effectively capture the memory limits for your deployment. `ROX_MEMLIMIT` sets the soft memory limit of the Go process to 95% of the configured amount. Define `ROX ...
```
```suggestion
* Use the `ROX_MEMLIMIT` environment variable, which replaces the `GOMEMLIMIT` variable in configuration files. Although you can still use the standard Go environment variable `GOMEMLIMIT`, you should use `ROX_MEMLIMIT` instead to capture the memory limits for your deployment more effectively. `ROX_MEMLIMIT` sets the soft memory limit of the Go process to 95% of the configured amount. Define `ROX_MEMLIMIT` as an integer without units representing the number of bytes.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
++
+For more information, see xref:../upgrading/upgrade-roxctl.adoc#upgrade-roxctl[Manually upgrading using the roxctl CLI].
+
+* Publish opensource instead of `stackrox.io` helm charts.
```
```suggestion
* Publish open source instead of `stackrox.io` helm charts.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+* The default policy `systemctl Execution` is not triggered when you use the `--version` process argument. This change does not lead to a security issue, as the printed information relates to the capabilities supported by `systemd` at the time of creation, and not the capabilities of your host operating system.
+
+* The default policy `No resource requests or limits specified` has been renamed to `No CPU request or memory limit specified`. The default policy no longer checks the CPU limit or memory request. Instead, it specifically recognizes whether the CPU request and memory limit are specified.
```
```suggestion
* The default policy `No resource requests or limits specified` has been renamed `No CPU request or memory limit specified`. It no longer checks the CPU limit or memory request; instead, it specifically recognizes whether the CPU request and memory limit are specified.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+* Configure the claim mappings of the authentication provider and required attributes from the UI.
+
+* Configure the API token expiration. If the expiration date is not specified, API token expires in 1 year.
```
```suggestion
* Configure the API token expiration date. If it is not specified, the API token expires in 1 year.
```

gaurav-nelson commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,572 @@
+
+* Configure the API token expiration. If the expiration date is not specified, API token expires in 1 year.
+
+* The information available on the *Network -> Listening Endpoints* page has been improved and updated, including the addition of new information about endpoints such as the pod UID and namespace, and the removal of endpoints from the deleted pods.
```
```suggestion
* The information available on the *Network -> Listening Endpoints* page has been improved and updated, including new information about endpoints such as the pod UID and namespace, and the removal of endpoints from the deleted pods.
```

gaurav-nelson reviewed (commented):
**praise:** 👏 Thank you for working on this thoroughly. I've marked many suggestions and a few issues. Feel free to contact me if something is not clear.

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,573 @@
+
+In {product-title-short} 4.4, selected connections to private IP addresses within the cluster, which were previously incorrectly identified as external entities, are now shown as internal entities in the Network graph.
+
+The following examples show scenarios where:
```
```suggestion
This fixes connections that are incorrectly identified as external, for example, when the following scenarios occur:
```

kcarmichael08 reviewed (commented):
LGTM

RTann commented on modules/find-clusters-with-most-kubernetes-and-istio-vulnerabilities.adoc:
```diff
@@ -1,22 +1,23 @@
 
 [role="_abstract"]
-Use the *Vulnerability Management* view for identifying the clusters with most Kubernetes and Istio vulnerabilities in your environment.
+Use the *Vulnerability Management (1.0)* view for identifying the clusters with most Kubernetes and Istio vulnerabilities (deprecated) in your environment.
```
OpenShift, too

RTann commented on modules/find-clusters-with-most-kubernetes-and-istio-vulnerabilities.adoc:
```diff
@@ -1,22 +1,23 @@
+Use the *Vulnerability Management (1.0)* view for identifying the clusters with most Kubernetes and Istio vulnerabilities (deprecated) in your environment.
 
-The *Clusters with most K8S & Istio Vulnerabilities*  widget shows a list of clusters, ranked by the number of Kubernetes and Istio vulnerabilities in each cluster.
+The *Clusters with most orchestrator and Istio vulnerabilities*  widget shows a list of clusters, ranked by the number of Kubernetes and Istio vulnerabilities (deprecated) in each cluster.
```
OpenShift, too

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,580 @@
+* In {product-title-short} 4.4, both the StackRox Scanner and the new Scanner V4 are available for scanning workloads, and the existing StackRox Scanner reports node and platform vulnerabilities.
+* {product-title-short} upgrades and new installations use the StackRox Scanner by default.
+* You now have the option to enable the new Scanner V4 in addition to the default StackRox Scanner. Scanner V4 is specifically designed for scanning images, while StackRox Scanner is still required for scanning nodes and platforms. This offers you additional benefits and an extended scope for securing your environment.
+* Red Hat plans to make the new Scanner V4 the default Scanner in future releases.
```
in a future release

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,580 @@
+[id="resolved-in-version-440_{context}"]
+=== Resolved in version 4.4.0
+
+*Release date*: 18 March 2023
```
2024

RTann commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,580 @@
+|Scanner DB
+|Stores image scan results and vulnerability definitions.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}`
+
```
Should also include the following:

```
|Scanner V4
|Scans images.
a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-v4-rhel8:{rhacs-version}`

|Scanner V4 DB
|Stores image scan results and vulnerability definitions for Scanner V4.
a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-v4-db-rhel8:{rhacs-version}`
```

RTann replied:
@msugakov @daynewlee can you help confirm these are the right names?

daynewlee replied:
based on Errata tool, that looks right:
https://errata.engineering.redhat.com/errata/container/128686

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -0,0 +1,588 @@
+[id="integration-with-red-hat-openshift-cluster-manager-and-paladin-cloud-to-discover-unsecured-clusters_{context}"]
+=== Integration with {cloud-redhat-com} and Paladin Cloud to discover unsecured clusters
+
+In {product-title-short} 4.4, you can now discover new clusters not protected by {product-title-short} through integration with {cloud-redhat-com} and Paladin Cloud. This feature provides a list of clusters in your Openshift environment or on cloud platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE) and Microsoft Azure Kubernetes Service (Micros ...
```
```suggestion
In {product-title-short} 4.4, you can now discover new clusters not protected by {product-title-short} through integration with {cloud-redhat-com} and Paladin Cloud. This feature provides a list of clusters in your OpenShift environment or on cloud platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE) and Microsoft Azure Kubernetes Service (Microsoft AKS) and provides a discovery mechanism to improve your organization's security.
```

RTann reviewed (approved):
LGTM from scanner perspective

kcarmichael08 commented:
/cherrypick rhacs-docs-4.4
